src/sonic-ctrmgrd/ctrmgr/container_state.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python3 -u
+
+import argparse
+import datetime
+import inspect
+import subprocess
+import syslog
+import time
+
+from swsscommon import swsscommon
+
+# DB field names
+SET_OWNER = "set_owner"
+
+CURRENT_OWNER = "current_owner"
+UPD_TIMESTAMP = "update_time"
+DOCKER_ID = "container_id"
+REMOTE_STATE = "remote_state"
+VERSION = "container_version"
+SYSTEM_STATE = "system_state"
+
+KUBE_LABEL_TABLE = "KUBE_LABELS"
+KUBE_LABEL_SET_KEY = "SET"
+KUBE_LABEL_UNSET_KEY = "UNSET"
+
+state_db = None
+set_owner = "local"
+state_data = {
+        CURRENT_OWNER: "none",
+        UPD_TIMESTAMP: "",
+        DOCKER_ID: "",
+        REMOTE_STATE: "none",
+        VERSION: "0.0.0",
+        SYSTEM_STATE: ""
+        }
+
+
+def debug_msg(m):
+    msg = "{}: {}".format(inspect.stack()[1][3], m)
```

Comment:
what is this inspect.stack()[1][3]? not sure I understand this.

Reply:
Captures the function name of the caller of "debug_msg".